compiler/src/dotty/tools/dotc/typer/RefChecks.scala

```diff
@@ -143,7 +143,7 @@ object RefChecks {
    *    1.8.1  M's type is a subtype of O's type, or
    *    1.8.2  M is of type []S, O is of type ()T and S <: T, or
    *    1.8.3  M is of type ()S, O is of type []T and S <: T, or
-   *    1.9    M must not be a typelevel def or a Dotty macro def
+   *    1.9    M must not be an erased definition
    *    1.10.  If M is a 2.x macro def, O cannot be deferred unless there's a concrete method overriding O.
    *    1.11.  If M is not a macro def, O cannot be a macro def.
```

Comment:
The `1.11` rule should probably be for `erased` in general.

Reply:
In fact, 1.11 is not needed at all, since macros are erased, hence effectively final.